source/client-side-operations-timeout/client-side-operations-timeout.md

```diff
@@ -0,0 +1,906 @@
+# Client Side Operations Timeout
+
+- Status: Accepted
+- Minimum Server Version: 2.6
+
+______________________________________________________________________
+
+## Abstract
+
+This specification outlines a new `timeoutMS` option to govern the
+amount of time that a single operation can execute before control is
+returned to the user. This timeout applies to all of the work done to
+execute the operation, including but not limited to server selection,
+connection checkout, and server-side execution.
```

Comment:
Did you plan on enforcing the line length with linting tools? Knowing how fast and loose folks are when editing specs (I've seen entire paragraphs written without line breaks), I think it'd be a matter of time before the line lengths get inconsistent.

Also, any thoughts on 120 vs. 80? When link syntax is involved, the 80-column limit results in some very aggressive wrapping.

Reply:
Line length is not enforceable with `mdformat`.